source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/organization-projects/table.tsx

```diff
@@ -34,7 +36,7 @@ export default compose(
   }),
   withNetwork(),
   withLoader(({ error, projects }) => !error && !projects),
-  withCache(({ applyFilter }) => ({
-    projects: q => applyFilter(q.findRecords(PROJECT), true)
+  withProps(({ projects }) => ({
+    projects: projects.filter(resource => resource.type === PROJECT)
```

Comment:
need to figure out why the payload for projects includes things that aren't projects. But, that is a thing to work on later

Reply:
+1